### DISCUSSION TITLE
Addiction API endpoints for project environments.

### DISCUSSION THREAD

**Discussion by char-ptr** [coollabsio/coolify#4144] [Feature Requests]
Currently there exists endpoints for getting project environments but no such endpoints exist for creating, or deleting.

Here in the source shows there exists only getting the environments associated with the project 
https://github.com/coollabsio/coolify/blob/main/app/Http/Controllers/Api/ProjectController.php#L103-L159

I think it would be beneficial to include such endpoints for further extensibility.

**Comment by nightvisi0n** [comment]
PR: https://github.com/coollabsio/coolify/pull/6255
